source/core/ib_stencils.cc

```diff
@@ -25,34 +25,29 @@ std::vector<double>
 IBStencils<dim>::coefficients(unsigned int order)
 {
   // Initialize the coefficient vector
-  std::vector<double> coef;
+  std::vector<double> coef(order+1);
```

Comment:
Be careful though. Here you should  not make coef order +1 if the order is more than 4 no? In this case, better to resize in if.

Reply:
i put a resize only in the case of order 5 or more